docs/libcurl/symbols-in-versions

```diff
@@ -570,6 +570,7 @@ CURLOPT_SSL_SESSIONID_CACHE     7.16.0
 CURLOPT_SSL_VERIFYHOST          7.8.1
 CURLOPT_SSL_VERIFYPEER          7.4.2
 CURLOPT_SSL_VERIFYSTATUS        7.41.0
+CURLOPT_SSL_SESSION_FILE        7.58.0
```

Comment:
FYI: the feature window for 7.58.0 is closed so it will have to get into a later version, possibly 7.59.0.

Reply:
fixed.